src/Orchard.Cms.Web/Modules/Orchard.OpenId/Views/Access/Authorize.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@using Orchard.OpenId.ViewModels
+@model AuthorizeViewModel
+
+<div class="jumbotron">
+    <h1>Authorization</h1>
+    
+    <p class="lead text-left">@T["Do you want to grant <strong>{0}</strong> access to your data? (scopes requested: {1})", @Model.ApplicationName, @Model.Scope]</p>
+
+    <form method="post">
+        @Html.AntiForgeryToken()
+
+        <input type="hidden" name="request_id" value="@Model.RequestId" />
```

Comment:
@jersiovic ah, I thought you meant the `[BindNever]` attribute, not the name of the parameter itself.

Yeah, you can't change it: it must be exactly `request_id`.